src/pages/VenuePage/VenuePage.tsx

```diff
@@ -277,7 +277,8 @@ const VenuePage: React.FC = () => {
       !event ||
       !venue ||
       !userPurchaseHistoryRequestStatus ||
-      !isVenueUsersLoaded
+      // @debt since we are not using users directly here, it would be nice to eventually remove this check from here
+      !isUniverseUsersLoaded
```

Comment:
What would be needed to remove this check from here now? What relies on it still being present at the moment?

Reply:
Will need to look at `users` usage all the way down in the tree to answer it

Reply:
Makes sense. May be worth looking into as part of this PR, and then either fixing here if it's trivial, or backlogging with context/details of what would be required to solve it if it's a bigger/more complex issue.

Reply:
Have removed that